### PR TITLE
modify prepostscripts according to bug 347

### PR DESCRIPTION
**Pull request by junxiawang**
modify prepostscripts file according to bug 347

### CODE REVIEW

**Comment by whowutwut** [comment]
@junxiawang  you need to put a # before 347 to make the GitHub reference, you can edit the post above to add the # so that we can have a reference to the issue...

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/prepostscripts/post_script.rst [hunk 1]
```diff
@@ -89,7 +89,7 @@ When your script is executed on the node, all the attributes in the site table a
 
 To run the postscripts, a script is built, so the above exported variables can be input. You can usually find that script in /xcatpost on the node and for example in the Linux case it is call mypostscript. A good way to debug problems is to go to the node and just run mypostscript and see errors. You can also check the syslog on the Management Node for errors.
 
-When writing you postscripts, it is good to follow the example of the current postscripts and write errors to syslog and in shell. See Suggestions for writing scripts.
+When writing you postscripts, it is good to follow the example of the current postscripts and write errors to syslog and in shell. See :ref:`Suggestions-for-write-scripts-label` .
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/prepostscripts/post_script.rst]: This section was copied over from SourceForge directly :-(   The fix here is not to create a link to a blank page, but to see whether the content is even worth bringing over to RTD. 

Here's the original page... 
http://sourceforge.net/p/xcat/wiki/Postscripts_and_Prescripts/#suggestions-for-writing-scripts

I think those tips are not very helpful without any examples.  I would suggest removing the link here...

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/prepostscripts/suggestions.rst [hunk 2]
```diff
@@ -1,3 +1,5 @@
+.. _Suggestions-for-write-scripts-label:
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/prepostscripts/suggestions.rst]: Suggest removing this page.  Add it back in later if we feel it's necessary.  Right now I don't think that "Suggestions for writing scripts" page over on SourceForge is adding anything valuable to the documentation.

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/prepostscripts/post_script.rst [hunk 3]
```diff
@@ -216,7 +216,7 @@ Example of mypostscript  ::
     run_ps script2
     # postscripts-end-here\n
 
-The mypostscript file is generated according to the mypostscript.tmpl file.
+The mypostscript file is generated according to the mypostscript.tmpl file and template details will show in below section.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/prepostscripts/post_script.rst]: I don't know how this improves the understanding...

**Comment by whowutwut** [comment]
@junxiawang , I don't think we should merge this pull request.  There are 6 items in @caomengmeng Issue #347  and the changes here only attempt to resolve 2 of the 6.. what about the other 4?  Closing for now, please re-open when the changes are more complete.

**Comment by tingtli** [comment]
Yes. I agree. Let's trace it in 2.12

**Pull request closed without merging**